topics/statistics/tutorials/fine_tuning_protTrans/tutorial.md

```diff
@@ -0,0 +1,109 @@
+
+
+## JupyterLab in Galaxy Europe
+
+
+### Open JupyterLab
+
+> <hands-on-title>GPU-enabled Interactive Jupyter Notebook for Machine Learning</hands-on-title>
+>
+> - {% tool [GPU-enabled Interactive Jupyter Notebook for Machine Learning](interactive_tool_ml_jupyter_notebook) %}
+>    - *"Do you already have a notebook?"*: `Start with a code repository`
+>    - *"Online code repository (Git-based) URL"*: `https://github.com/anuprulez/fine-tune-protTrans-repository` 
+>    - Click *"Run Tool"*
+>
+>    > <comment-title></comment-title>
+>    > The above step automatically fetches the notebook and datasets from the provided GitHub URL and initiates a JupyterLab.
+>    >  If you do not have access to this resource in Galaxy Europe, please apply for it at: [Access GPU-JupyterLab](http://usegalaxy.eu/gpu-request). It may take a day or two to receive access.
+>    >
+>    {: .comment}
+{: .hands_on}
+
+
+## Fine-tuning notebook
+From the cloned repository, open the `fine-tune-protTrans-dephophorylation.ipynb` notebook. The notebook contains all the necessary scripts for processing protein sequences, creating and configuring protein large language models, training it on the protein sequences evaluating them on the test protein sequences and visualising results. Let's look at these key steps of fine-tuning.
+
+
+### Install necessary Python packages
+The protein large language model has been developed using Pytorch and the model weights are stored at HuggingFace. Therefore, packages such as Pytorch, Transformers, and SentencePiece must be installed in the notebook to recreate the model. Additional packages such as Scikit-learn, Pandas, Matplotlib and Seaborn are also required for data preprocessing, manipulation and visualisation of model training and test performances. 	All the necessary packages are installed in the notebook using `!pip install` command. Note: the installed packages have a lifespan equal to the notebook sessions. When a new session of JupyterLab is created, all the packages need to be installed again.
+
+### Fetch and split data
+After installing all the packages and importing necessary Python packages, protein sequences (available as a FASTA file) and their labels are read into the notebook. These sequences are further divided into training and validation sets. The training set is used for fine-tuning the protein large language model, and the validation set is used for model evaluation after each training epoch.
```

Comment:
Instead of:
`After installing all the packages and importing necessary Python packages`
try
`After installing and importing all the necessary packages`